features/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_TFM/TARGET_TFM_V1_0/VERSION.txt

```diff
@@ -1 +1 @@
-eecaad9e4014
+9fd2fd6250d5
```

Comment:
Not sure that's intentional?

Reply:
Yes. Intentional. Version is updated along with the updated TFM binary